pylint/checkers/classes.py

```diff
@@ -1221,8 +1228,13 @@ def is_first_attr(self, node):
         """Check that attribute lookup name use first attribute variable name
         (self for method, cls for classmethod and mcs for metaclass).
         """
-        return self._first_attrs and isinstance(node.expr, astroid.Name) and \
-                   node.expr.name == self._first_attrs[-1]
+        return self.is_first_name(node.expr)
+
+    def is_first_name(self, node):
+        """Check if astroid.Name corresponds to first attribute variable name
+        (self for method, cls for classmethod and mcs for metaclass)."""
```

Comment:
Add an empty line between these two and let's put the closing quotes on the next line.